source/includes/harvest/_candidates.md

```diff
@@ -1310,3 +1310,129 @@ On-Behalf-Of | ID of the user issuing this request. Required for auditing purpos
 Parameter | Required | Type | Description
```

Comment:
Not sure why GitHub thinks there's something wrong with the mardown. It looks fine in Slate. Sorry that it's red.